test/unit/env_nvim2_spec.lua

```diff
@@ -0,0 +1,23 @@
+local helpers = require("test.unit.helpers")
+local cimport = helpers.cimport
+local env = cimport('./src/nvim/os/os.h')
+local eq = helpers.eq
+local neq = helpers.neq
+
+describe('Dont use $VIM/$VIMRUNTIME', function()
+
+  env.os_unsetenv('NVIM')
+  env.os_unsetenv('NVIMRUNTIME')
+  env.os_setenv('VIM', 'VIM_VALUE', 1)
+  env.os_setenv('VIMRUNTIME', 'VIMRUNTIME_VALUE', 1)
+
+  helpers.vim_init()
```

Comment:
Is this why the tests need to be separate? Or can they be combined to 1 env_nvim_spec.lua file? I am also wondering why there are nested `describe()` blocks with no `it()` specs. Suggestion:

```lua
local helpers = require("test.unit.helpers")
local cimport = helpers.cimport
local env = cimport('./src/nvim/os/os.h')
local eq = helpers.eq
local neq = helpers.neq

describe('$NVIM/$NVIMRUNTIME', function()
  before_each(function()
    env.os_unsetenv('NVIM')
    env.os_unsetenv('NVIMRUNTIME')
    env.os_unsetenv('VIM')
    env.os_unsetenv('VIMRUNTIME')

    helpers.vim_init()
  end)

  it('$VIM is cleared if $NVIM is unset', function()
    env.os_setenv('VIM', 'VIM_VALUE', 1)
    neq(os.getenv('VIM'), 'VIM_VALUE')
  end)
  it('$VIMRUNTIME is cleared if $NVIMRUNTIME is unset', function()
    env.os_setenv('VIMRUNTIME', 'VIMRUNTIME_VALUE', 1)
    neq(os.getenv('VIMRUNTIME'), 'VIMRUNTIME_VALUE')
  end)

  it('$VIM is set to $NVIM', function()
    env.os_setenv('NVIM', 'NVIM_VALUE', 1)
    eq(os.getenv('VIM'), os.getenv('NVIM'))
  end)
  it('$VIMRUNTIME is set to $NVIMRUNTIME', function()
    env.os_setenv('NVIMRUNTIME', 'NVIMRUNTIME_VALUE', 1)
    eq(os.getenv('VIMRUNTIME'), os.getenv('NVIMRUNTIME'))
  end)
end)


```